src/components/Boards/lib/generateBoardName.ts

```diff
@@ -0,0 +1,9 @@
+export default function() {
```

Comment:
Is this redundant given `BoardContainer.tsx`?

Reply:
You mean just leaving this in inline in board container? Can go either way on this one, not opinionated one way or another.

Reply:
We can keep as is for now and if it makes sense to refactor later, we can do it